build.sbt

```diff
@@ -95,6 +95,16 @@ lazy val client = libraryProject("client")
     theDsl % "test->compile",
     scalaXml % "test->compile")
 
+lazy val clientMetrics = libraryProject("client-metrics")
+  .settings(
+    description := "Support for Dropwizard Metrics on the client",
+    libraryDependencies ++= Seq(
+      metricsCore,
+      metricsJson
```

Comment:
Do we use metrics-json, or is this a copy and paste from the server?  I think metrics-core is enough.

Reply:
yes, you are totally right. I've removed that dependency